lib/stat_tracker.rb

```diff
@@ -35,4 +38,78 @@ def game_teams_stats
       row.delete(:takeaways)
     end
   end
+
+  def teams_stats
+    teams_data = CSV.read(@teams, { encoding: 'UTF-8', headers: true, header_converters: :symbol, converters: :all })
+    hashed_teams_data = teams_data.map { |row| row.to_hash }
+    hashed_teams_data.each do |row|
+      row.delete(:staduim)
+    end
+  end
+
+  def highest_total_score
+    @game_statistics.highest_total_score
+  end
+
+  def lowest_total_score
+    @game_statistics.lowest_total_score
+  end
+
+  def percentage_home_wins
+    @game_statistics.percentage_home_wins
+  end
+
+  def percentage_visitor_wins
+    @game_statistics.percentage_visitor_wins
+  end
+
+  def percentage_ties
+    @game_statistics.percentage_ties
+  end
+
+  def count_of_games_by_season
+    @game_statistics.count_of_games_by_season
+  end
+
+  def average_goals_per_game
+    @game_statistics.average_goals_per_game
+  end
+
+  def average_goals_by_season
+    @game_statistics.average_goals_by_season
+  end
+
+  def count_of_teams
+    @league_statistics.count_of_teams
+  end
+
+  def best_offense
+    @league_statistics.best_offense_stats
+    @league_statistics.best_offense
+  end
+
```

Comment:
Are these methods an opportunity for modules?

Reply:
I think they absolutely are, we should look into that when refactoring!